lib/useDataApi.js

```diff
@@ -4,18 +4,16 @@ import unfetch from 'isomorphic-unfetch';
 const dataFetchReducer = (state, action) => {
     switch (action.type) {
         case 'FETCH_INIT':
-            return { ...state, isLoading: true, isError: false };
+            return { ...state, isError: false };
```

Comment:
It's a generic data fetching hook, it should keep the `isLoading` state, because no data does not always mean that fetching is ongoing

Reply:
Which case ? (API insipiration https://swr.now.sh/)  I can keep `isLoading` if you want

Reply:
Yeah, the hook has three states :
- Loading
- In error
- Loaded